easybuild/easyconfigs/m/MM-align/MM-align-1.0-goolf-1.4.10.eb

```diff
@@ -0,0 +1,32 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = 'CmdCp'
+
+name = 'MM-align'
+version = '1.0' # no version info provided. I decide it's 1.0
```

Comment:
or, make the above a -versionsuffix and use a date as a version (probably more broad scope, covers more needs)